msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,219 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards usually match the look and feel of the host application and have a fixed layout that remains consistent across Teams mobile and desktop clients.
+
+Adaptive Card responsive layout helps you to design Adaptive Cards with different layouts that adapt to different card widths in the context where the card is displayed. When you design Adaptive Card for your app, ensure that you provide a better user experience across chat, channels, and meeting chat in Teams desktop and mobile clients.
+
+[Add image for responsive Adaptive card in mobile and desktop.]
+
+#### Design responsive Adaptive Cards
+
+You can design a responsive Adaptive Card using the `targetWidth` property, which allows you to show or hide any element based on the card's width.
+
+You can set the `targetWidth` property to any element in your Adaptive Card to create different layouts for different card widths. The following table provides the available values of the `targetWidth` property for different layouts:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `veryNarrow` | Use for Adaptive Card on a desktop mode in a meeting chat. |
+  | `narrow` | Use for Adaptive Card on a mobile phone in portrait mode. |
+  | `standard` | Use for Adaptive Card  on a mobile phone in landscape mode, a tablet in portrait mode, or a chat in desktop mode. The default value is `standard`.|
+  | `wide` | Use for Adaptive Card on a tablet in landscape mode, or a chat or channels in desktop mode with the card width as `full width`. |
+
+The following images and JSON samples show an example of Adaptive Card designed without using `targetWidth` and using `targetWidth`:
```

Comment:
Below is an example of a card authored without using `targetWidth` and an updated version of the same card that does use `targetWidth`. This example illustrates how using `targetWidth` can make the layout of your card scale perfectly across card widths.